test/metabase/automagic_dashboards/names_test.clj

```diff
@@ -61,6 +61,18 @@
         (is (= "number of Venues where Name is Test"
                ;; Test specifically the un-normalized form (metabase#15737)
                (names/cell-title root ["=" ["field" %name nil] "Test"]))))
+      (testing "Should humanize greater than filter"
+        (is (= "number of Venues where Name is greater than Test"
+               (names/cell-title root [">" ["field" %name nil] "Test"]))))
+      (testing "Should humanize at least filter"
+        (is (= "number of Venues where Name is at least Test"
```

Comment:
I wonder if, since we make `>` become _"greater than"_ it might make sense to have
`>=` become _"greater than or equal to"_ . Subjectively, that's actually the phrase I use if I'm saying `>=` outloud anyway.

Not a strong opinion of mine, just a gut check.

Reply:
Thought about this as well. IDK. What would a nontechnical audience say? Easy to change if product would prefer more technical jargon.